department-of-veterans-affairs/GIBCT-Data-Service
# GIDS: Institution CSV is blank on export #6201

## Description
https://github.com/department-of-veterans-affairs/va.gov-team/issues/6201

Issue was that export of versioned tables was using `version.id` instead of `version.number` which is what is passed in to the method

```
SELECT * FROM "institutions" WHERE "institutions"."version_id" = $1  [["version_id", <version_number>]]
```
vs
```
SELECT * FROM "institutions" JOINS "versions" ON "institutions"."version_id" = "versions.id" WHERE "versions"."number" = $1  [["number", <version_number>]]
```
## Testing done
To test 
- generate a preview
- in SQL change the number value of the verisons row that was just created from preview generation so that id does not equal number for the row
- refresh dashboard, preview should now show the new "number" you set
- export the preview version

## Screenshots


## Acceptance criteria

## Definition of done
